linera-chain/src/unit_tests/data_types_tests.rs

```diff
@@ -11,7 +11,7 @@ use crate::test::{make_first_block, BlockTestExt};
 fn test_signed_values() {
     let key1 = KeyPair::generate();
     let key2 = KeyPair::generate();
-    let name1 = ValidatorName(key1.public());
+    let name1 = ValidatorPublicKey(key1.public());
```

Comment:
These renamings are always work work because we forget places like here, the variable.